app/workers/evss/disability_compensation_form/submit_form526_cleanup.rb

```diff
@@ -4,7 +4,7 @@ module EVSS
   module DisabilityCompensationForm
     class SubmitForm526Cleanup < Job
       include Sidekiq::Worker
-
+      # TODO: AEC
       FORM_ID = '21-526EZ'
```

Comment:
`21-526EZ` is version 1, and shouldn't be in use any more.

Reply:
@Mottie I'm pretty sure version 2 uses this ID as well. They share IDs.

Reply:
I verified that all the in progress 526 forms have id 21-526EZ (before we implement bed)